PlaygroundAppUITests/XCTWaiter.Result+CustomStringConvertible.swift

```diff
@@ -0,0 +1,15 @@
+import XCTest
+
+extension XCTWaiter.Result: CustomStringConvertible {
+
+    public var description: String {
+        switch self {
+        case .completed: return "XCTWaiter.Result .completed"
+        case .incorrectOrder: return "XCTWaiter.Result .incorrectOrder"
+        case .interrupted: return "XCTWaiter.Result .interruted"
+        case .invertedFulfillment: return "XCTWaiter.Result .invertedFulfillment"
+        case .timedOut: return "XCTWaiter.Result .timedOut"
+        @unknown default: return "XCTWaiter.Result unknown case – Please file a bug report at https://github.com/Automattic/XCUITestHelpers"
```

Comment:
`@unknonw default` is a Swift that allows `enum` creators (in this case the Apple team that built `XCTWaiter.Result` to declare that there might be more cases added in the future and require consumers of the code to handle them somehow.

More details in [the Swift feature proposal document](https://github.com/apple/swift-evolution/blob/main/proposals/0192-non-exhaustive-enums.md).